lib/operators/param.js

```diff
@@ -1,7 +1,7 @@
 var errors = require("../errors");
 
 module.exports = {
-  parse: function(key, dflt) {
+  parse: function() {
```

Comment:
Yeah that sounds better because its kind of like an interface which this function decides to ignore.